tests/browser/test/basic_playground_test.js

```diff
@@ -129,6 +136,27 @@ suite('Right Clicking on Blocks', function (done) {
     chai.assert.isTrue(isEnabled);
   });
 
+  test('Add Comment', async function () {
+    const block = await dragNthBlockFromFlyout(browser, 'Loops', 0, 20, 20);
+
+    const blockId = block.id;
```

Comment:
Since the `blockId` is only used once, I think it makes sense to just pass it to `getCommentText`!

Reply:
Done.